src/h5web/providers/mock/data.ts

```diff
@@ -103,70 +103,18 @@ export const mockMetadata = makeNxGroup(mockDomain, 'NXroot', {
       ],
     }),
     makeGroup('nexus_malformed', [
-      makeGroup('default_not_string', [], {
-        attributes: [makeAttr('default', scalarShape, intType, 42)],
-      }),
-      makeGroup('default_empty', [], {
-        attributes: [makeStrAttr('default', '')],
-      }),
```

Comment:
Removing a few unlikely cases, here and in the E2E tests, now that they are better covered with unit tests.